iphone/Maps/UI/PlacePage/Components/PlacePageHeader/PlacePageHeaderViewController.swift

```diff
@@ -50,7 +50,35 @@ extension PlacePageHeaderViewController: PlacePageHeaderViewProtocol {
     }
   }
 
-  func setTitle(_ title: String) {
-    titleLabel?.text = title
+  func setTitle(_ title: String?, secondaryTitle: String?) {
+    guard let title = title,
+          let titleFont = StyleManager.shared.theme?.fonts.medium20 else {
+      titleLabel?.attributedText = nil
```

Comment:
1. Previously, the `.text` property wasn't nil. It was either an empty or non-empty string. Now it is nil. Is it better/more robust?
2. Why attributed text is necessary at all? Why is it not possible to set the proper font for the label in .storyboard?

Reply:
1. I don't know why previous code has nil-coalescing to empty string `view?.setTitle(placePagePreviewData.title ?? "")`. It isn't needed because `UILabel`'s `text` property can be set to optional `String?`.
2. I desided to use attributed text because in this case it's possible to set two different fonts on first and second line. I'm not sure that it's possible to set in storyboard runtime attributes (but I don't have a lot of experience with storyboards preferring making all layout in code). And we don't need additional `UILabel` and layout logic with some `NSLayoutConstraint` properties to manage correct height of entire view. Because secondary title is optional it will be more code than with my decision.